LogMonitor/LogMonitorTests/EtwMonitorTests.cpp

```diff
@@ -203,7 +203,7 @@ namespace LogMonitorTests
             fflush(stdout);
 
 
-            std::function<void(void)> f1 = [&etwProviders] { EtwMonitor etwMonitor(etwProviders, L"json"); };
+            std::function<void(void)> f1 = [&etwProviders] { EtwMonitor etwMonitor(etwProviders, L"json", L""); };

```

Comment:
nit: could the second param just be optional with `""` default, instead of passing it here?

Reply:
@profnandaa It was a trade-off between creating an overloaded function that takes 2 parameters or assigning default value to CustomLogFormat.

Reply:
Sorry, I meant modifying Line 35 of EtwMonitor constructor.